styles/style.css

```diff
@@ -46,6 +47,11 @@ nav li {
     color: black;
 }
 
+#intro a:hover {
```

Comment:
Good job on creating interaction for the call of action, as the hover allows an easy visualization for the user